invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/accordion.overrides

```diff
@@ -76,10 +76,11 @@ div.affiliations.accordion div.title button.down {
     }
   }
 
-  .panel-heading {
-    border-radius: 10px;
+  .title:not(.ui).panel-heading {
+    border-radius: @defaultBorderRadius;
     background-color: @mutedAccordionHeaderColor;
     color: @primaryColor !important;
+    padding: 0.78571429em 0.78571429em; // matching SUI's padding on <td> elements
```

Comment:
the value of pdding should come from SUI variable
https://github.com/Semantic-Org/Semantic-UI-LESS/blob/master/definitions/collections/table.less#L132

Reply:
Ahh! Thanks! I found it hardcoded somewhere else. Will update

Reply:
if it was hardcoded,  please update the hardcoded one too

Reply:
it was somewhere in the SUI repos